x-pack/docs/en/security/auditing/enable-audit-logging.asciidoc

```diff
@@ -3,21 +3,28 @@
 == Enabling audit logging
 
 You can log security-related events such as authentication failures and refused connections
-to monitor your cluster for suspicious activity. 
+to monitor your cluster for suspicious activity (including data access authorization and user
+security configuration changes).
+
 Audit logging also provides forensic evidence in the event of an attack.
 
 [IMPORTANT]
 ============================================================================
 Audit logs are **disabled** by default. You must explicitly enable audit logging.
 ============================================================================
+--
+TIP: Audit logs are only available on certain subscription levels.
+For more information, see https://www.elastic.co/subscriptions.
```

Comment:
Not a mandatory change, but in case this information moves, we've got an attribute for it:
```suggestion
For more information, see {subscriptions}.
```